dspace-server-webapp/src/test/java/org/dspace/app/rest/matcher/ExternalSourceMatcher.java

```diff
@@ -22,7 +23,9 @@ private ExternalSourceMatcher() {
         return allOf(
             hasJsonPath("$.id", is(id)),
             hasJsonPath("$.name", is(name)),
-            hasJsonPath("$.hierarchical", is(hierarchical))
+            hasJsonPath("$.hierarchical", is(hierarchical)),
+            hasJsonPath("$._links.entries.href", is(REST_SERVER_URL +
+                                                            "integration/externalsources/" + name + "/entries"))
```

Comment:
this actually demonstrate the existence of the bug and the solution, so if we wan't to deal with URI template this PR can be accepted